src/Adapter/Service/AntifraudService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Adapter\Service;
+
+use App\Domain\Account\Transaction\AntifraudServiceInterface;
+use GuzzleHttp\Client;
+
+final class AntifraudService implements AntifraudServiceInterface
+{
+    private Client $client;
+    private string $uri;
+
+    public function __construct(Client $client, string $uri)
+    {
+        $this->client = $client;
+        $this->uri = $uri;
+    }
+
+    public function authorize(): bool
+    {
+        $res = $this->client->request('GET', $this->uri);
```

Comment:
`$res` is the best name for it?

https://www.codementor.io/@veenitchauhan/basics-of-naming-conventions-for-php-developers-eliexmew6